docs/contributing/provider_development_guide.md

```diff
@@ -652,6 +652,6 @@ feedback you may have about the development process!
 [Crossplane CLI]: https://github.com/crossplane/crossplane-cli#quick-start-stacks
 [`angryjet` documentation]: https://github.com/crossplane/crossplane-tools/blob/master/README.md
 [code generation guide]: https://github.com/crossplane-contrib/provider-aws/blob/master/CODE_GENERATION.md
-[Terrajet]: https://github.com/crossplane/terrajet
-[Generating a Crossplane Provider guide]: https://github.com/crossplane/terrajet/blob/main/docs/generating-a-provider.md
+[Terrajet]: https://github.com/upbound/upjet
```

Comment:
Appears this link is broken:
```suggestion
[Upjet]: https://github.com/upbound/upjet
```

Reply:
Ping @luebken. This needs an update before we can merge.